src/plugins/wizard/public/application/utils/use/use_saved_wizard_vis.ts

```diff
@@ -83,14 +100,12 @@ export const useSavedWizardVis = (visualizationIdFromUrl: string | undefined) =>
               mapping: managementRedirectTarget,
             })(error);
           }
+          if (error instanceof InvalidJSONProperty) {
```

Comment:
Whats the user experience if in this error path? Since we dont redirect, what will the user see here?

Reply:
Is there any page that we should redirect to here? currently user will see a tooltip saying which property is invalid and what the correct type of the property is supposed to be, and an empty edit wizard page
<img width="1718" alt="Screen Shot 2022-09-14 at 1 36 29 PM" src="https://user-images.githubusercontent.com/43937633/190257445-96f1ea57-8315-4090-943f-a1dcbff344be.png">

Reply:
I'm okay with that. Can we use an error toast instead of a warning though?

Reply:
Sure, but i see currently in the code, fail to load visualization is using a warning toast as well. Do we want to change that to an error? https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/wizard/public/application/utils/use/use_saved_wizard_vis.ts#L88

Reply:
checked with the visualization plugin, and it's using addDanger to show an error toast for 'fail to load visualization', so i think we can use addDanger here for both scenario